src/legacy/server/status/routes/api/register_stats.js

```diff
@@ -50,6 +51,18 @@ export function registerStatsApi(usageCollection, server, config) {
     return usageCollection.toObject(usage);
   };
 
+  let lastMetrics = null;
+  /* kibana_stats gets singled out from the collector set as it is used
+   * for health-checking Kibana and fetch does not rely on fetching data
+   * from ES */
+  server.newPlatform.setup.core.metrics.getOpsMetrics$().subscribe(_metrics => {
+    const metrics = cloneDeep(_metrics);
+    lastMetrics = {
+      ...metrics,
+      timestamp: new Date().toISOString(),
+    };
```

Comment:
NIT: are you altering the `_metrics` nested properties in any way further down, or only consolidating it with new properties? If only consolidating, the spread should be enough and `cloneDeep` can probably be removed.

Reply:
Done